llama_stack/apis/post_training/post_training.py

```diff
@@ -111,11 +116,7 @@ class PostTrainingSFTRequest(BaseModel):
     validation_dataset_id: str
 
     algorithm: FinetuningAlgorithm
-    algorithm_config: Union[
-        LoraFinetuningConfig, QLoraFinetuningConfig, DoraFinetuningConfig
-    ]
-
-    optimizer_config: OptimizerConfig
+    algorithm_config: LoraFinetuningConfig
```

Comment:
when we add a full-sft, we'd need a FullFinetuningConfig also right? in that case, shouldn't we make this look like:

```
AlgorithmConfig = Annotated[
   Union[LoraFinetuningConfig, FullFinetuningConfig],
   Field(discriminator="type")
]

algorithm_config: AlgorithmConfig
```

?

Reply:
I don't think full sft has its specific params to pass